src/Subnet/IPv4Subnet.php

```diff
@@ -47,4 +46,9 @@ public function byteArray(): array
     {
         return $this->subnet->byteArray();
     }
+
+    public function contains(SubnetInterface $subnet): bool
+    {
+        return $subnet->prefix() >= $this->prefix();
```

Comment:
Very nice, I would do this in the contains function of the address but this is way nicer. One extra check on the IP family is required in this case to make it 'correct'. With your implementation an IPv6/32 would contain an IPv4/32. Which isn't the case.

Reply:
Done